bin/cli.js

```diff
@@ -39,6 +41,19 @@ const browserLaunchers = {
 
 const rmDuplicates = arr => Array.from(new Set(arr));
 
+const merge = (arr1, arr2) => (arr1 ? arr1.concat(arr2) : arr2);
+
+const mergeUnique = (arr1, arr2) => rmDuplicates(merge(arr1, arr2));
```

Comment:
I'd prefer to use `common.Iterator`:
```js
[...iter(arr1).join(arr2).collectTo(Set)]
```

Reply:
But here `merge` handles null/undefined `arr1` argument and your suggestion does not, or I am misunderstanding?
Though if we can remove that case from those functions then yes, I'm also in favor of @nechaido suggestion.

Reply:
@lundibundi 
```hs
const merge = (arr1 = [], arr2 = []) => [...iter(arr1).join(arr2).collectTo(Set)]`
```